ui/imports/utils/Global.qml

```diff
@@ -29,6 +28,7 @@ QtObject {
 
     signal openProfilePopupRequested(string publicKey, var parentPopup, bool openNicknamePopup)
     signal openChangeProfilePicPopup()
+    signal displayToastMessage(string title, string subTitle, string icon, bool loading, int ephNotifType, string url)
```

Comment:
I saw there are a lot of usages of normal ephNotifType  type w/o subtitle and icon. Did you think about another signal with less number of parameters ? I do not have a strong opinion though.

Reply:
@endulab since those toast messages will be globally accessible and used for/in many different cases we need to have them easy configurable and that's possible if we expose all necessary parameters which may be set for that component.